examples/function_memory_game/README.md

```diff
@@ -0,0 +1,15 @@
+# rust-memory-game
+
+
+A port version of [react-memory-game](https://github.com/leftstick/react-memory-game). You are able to try it at [here](https://img.shields.io/website?label=demo&url=https%3A%2F%2Fexamples.yew.rs%2Ffunction_memory_game)](https://examples.yew.rs/function_memory_game)
+
+## Concepts
+
+- Uses [`function_components`](https://yew.rs/docs/next/concepts/function-components)
+- Uses [`gloo_storage`](https://gloo-rs.web.app/docs/storage) to persist the state
+
+
+
+## LICENSE
+
+[MIT License](https://raw.githubusercontent.com/yewstack/yew/examples/function_memory_game/master/LICENSE)
```

Comment:
Why is the license specified here? Why is it only MIT? The example's Cargo.toml says the license is MIT to Apache 2.0.

Reply:
@hamza1311 this example should not have a specific license. I will remove this part